src/models/transactions/common.ts

```diff
@@ -24,7 +24,7 @@ const transactionTypes = [
   "TrustSet",
 ];
 
-const isMemo = (obj: { Memo: Memo }): boolean => {
+function isMemo(obj: { Memo: Memo }): boolean {
```

Comment:
I think this needs to be re-rebased on top of `2.0` - I made a lot of changes here when I linted `src/models`